com.microsoft.mrtk.accessibility/Utilities/DescribableObject.cs

```diff
@@ -12,6 +12,15 @@ namespace Microsoft.MixedReality.Toolkit.Accessibility
     [AddComponentMenu("MRTK/Accessibility/Describable Object")]
     public class DescribableObject : MonoBehaviour
     {
+        [SerializeField]
+        [Tooltip("What is the classification (ex: person, place, ui element, etc.) is this object?")]
+        private ObjectClassification classification = ObjectClassification.Things;
```

Comment:
By default, _can_ I select more than one classification here, and I just need to make sure I don't?

Reply:
That is true. If we opt to requiring the caller (screen reader) to make multiple calls (one per classification) we can avoid the issue,